runners/jet/src/main/java/org/apache/beam/runners/jet/JetRunner.java

```diff
@@ -186,7 +189,15 @@ private JetInstance getJetInstance(JetPipelineOptions options) {
   }
 
   private static List<PTransformOverride> getDefaultOverrides() {
-    return Collections.emptyList();
+    return Arrays.asList(
```

Comment:
Is this list exhaustive? How can I tell if something is missing?

Reply:
There is no exhaustive list as each runner can choose to implement each transform however they wish. We have to rely on the `@ValidatesRunner` suite of tests for validation.